examples/browser-create-react-app/README.md

```diff
@@ -1,15 +1,15 @@
 ## IPFS React app
 
-A minimal demonstration of how to use js-ipfs in a `create-react-app` generated app.
+This is a minimal demonstration of how to use `js-ipfs` in a `create-react-app` generated app.
 
-It boots up a js-ipfs instance (an IPFS node) via a custom React hook in `./src/hooks/use-ipfs-factory.js`, which is called from `./src/App.js`. Once the IPFS node is set up, `App.js` displays its ident and its version number.
+It boots up a `js-ipfs` instance (an IPFS node) via a custom React hook in `./src/hooks/use-ipfs-factory.js`, which is called from `./src/App.js`. Once the IPFS node is set up, `./src/App.js` displays the ident of this node and the version number of `js-ipfs` used to spawn it.
 
-> _Remember that a Peer ID of an IPFS node is [the multihash of the public key of this node](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#peer-ids), and the public-private key pair of a node is generated by typing `ipfs init`._
+> _Side note: what we call ident here, or "id", is the Peer ID of the IPFS node and is [the multihash of the public key of this node](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#peer-ids). The public-private key pair of a node is generated by typing `ipfs init`._
```

Comment:
This is a bit weird, I don't think we refer to a peer id as an "ident" anywhere else.

```suggestion
> _Side note: The PeerId of the IPFS node is [the multihash of the public key of this node](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#peer-ids). The public-private key pair of a node is generated by typing `ipfs init`._
```